code/modules/reagents/newchem/drinks.dm

```diff
@@ -154,4 +173,48 @@
 	mix_message = "The ingredients mix into a dark brown substance."
 	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
 
+/datum/reagent/ethanol/synthanol/uplink
+	name = "Uplink"
+	id = "uplink"
+	description = "A potent mix of alcohol and synthanol. Will only work on synthetics."
+	reagent_state = LIQUID
+	color = "#e7ae04"
+
+/datum/chemical_reaction/synthanol/uplink
+	name = "Uplink"
+	id = "uplink"
+	result = "uplink"
+	required_reagents = list("rum" = 1, "vodka" = 1, "tequilla" = 1, "whiskey" = 1, "synthanol" = 1)
+	result_amount = 5
+	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
+
+/datum/reagent/ethanol/synthanol/synthnsoda
+	name = "Synth 'n Soda"
+	id = "synthnsoda"
+	description = "The classic drink adjusted for a robot's tastes."
+	reagent_state = LIQUID
+	color = "#7204e7"
+
+/datum/chemical_reaction/synthanol/synthnsoda
+	name = "Synth 'n Soda"
+	id = "synthnsoda"
+	result = "synthnsoda"
+	required_reagents = list("synthanol" = 1, "cola" = 1)
+	result_amount = 2
+	mix_sound = 'sound/goonstation/misc/drinkfizz.ogg'
```

Comment:
Perhaps you could move this up into the `/datum/chemical_reaction/synthanol` class, so you don't have to constantly re-define it?